Documentation/ceph-cluster-crd.md

```diff
@@ -317,14 +345,30 @@ If you want to learn more, please read
 
 Based on the configuration, the operator will do the following:
 
-  1. if only the `public` selector is specified both communication and replication will happen on that network
-  2. if both `public` and `cluster` selectors are specified the first one will run the communication network and the second the replication network
+  1. If only the `public` selector is specified communication will happen on that network
+```yaml
+  network:
+    provider: multus
+    selectors:
+      public: rook-ceph/rook-public-nw
+```
+  2. If only the `cluster` selector is specified OSD heartbeat, object replication and recovery will happen on that network.
```

Comment:
Where is all the public traffic? On the default CNI, right? Also it would be good to define the internal traffic as a separate note so that bullet 3 can also reference it
```suggestion
  2. If only the `cluster` selector is specified, the internal cluster traffic* will happen on that network. All other traffic to mons, OSDs, and other daemons will be on the default network.

* Internal cluster traffic includes OSD heartbeats, data replication, and data recovery
* ```

Reply:
updated